packages/xss_filters/xss_filters.0.1/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "1.2"
+maintainer: "Edgar Aroutiounian <edgar.factorial@gmail.com>"
+authors: "Edgar Aroutiounian <edgar.factorial@gmail.com>"
+homepage: "https://github.com/bean-code/ocaml-npm-xss-filters"
+bug-reports: "https://github.com/bean-code/ocaml-npm-xss-filters/issues"
+license: "BSD-3-clause"
+dev-repo: "https://github.com/bean-code/ocaml-npm-xss-filters.git"
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+build-test: [
+  ["ocaml" "setup.ml" "-configure" "--enable-tests"]
+  ["ocaml" "setup.ml" "-build"]
+  ["ocaml" "setup.ml" "-test"]
+]
+remove: ["ocamlfind" "remove" "xss_filters"]
+depends: [
+  "nodejs" {>= "0.5"}
+  "ocamlfind" {build}
+]
+post-messages: [
+  "Sanitize user input with Yahoo's xss-filters npm package"
+  "
+open Nodejs
```

Comment:
This should not be here but rather in the `.mli` — the post message is displayed once, the `.mli` is where one goes for documentation.